sktime/forecasting/base.py

```diff
@@ -173,6 +333,31 @@ def _set_now(self, now=None):
     def _set_fh(self, fh):
         raise NotImplementedError
 
+    def _get_relative_fh(self):
```

Comment:
This should be:
```suggestion
    def _get_absolute_fh(self):
```
as our current `fh` is relative to the end/start of training set.